.drone.star

```diff
@@ -284,6 +292,27 @@ def uploadCoverage(ctx):
           },
         },
       },
+      {
+        'name': 'show-after',
+        'image': 'owncloud/alpine:latest',
+        'pull': 'always',
+        'commands': [
+          'cat coverage/*',
```

Comment:
Do you want to keep this "debugging" or or not?

Reply:
removed